src/MongoDB.Driver.Legacy/CommandResults/MapReduceResult.cs

```diff
@@ -92,6 +93,7 @@ public TimeSpan Duration
         /// <summary>
         /// Gets the emit count.
         /// </summary>
+        [Obsolete("EmitCount is ignored by server versions 4.3.1 and newer.")]
```

Comment:
Slight rewording requested:

    [Obsolete("EmitCount is not available on server versions 4.3.1 and newer.")]

Reply:
done